docs/components/networktransform.md

```diff
@@ -51,8 +51,33 @@ darkImageSrc="/img/BufferedTick_Dark.png?text=DarkMode"/>
 
 `NetworkTransform` always synchronizes positions from the server to the clients and position changes on the clients are not allowed. Netcode for GameObjects comes with a sample containing a `ClientNetworkTransform`. This transform synchronizes the position of the owner client to the server and all other client allowing for client authoritative gameplay.
 
-The `ClientNetworkTransform` lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`
 
-Or you can directly add this line to your `manifest.json` file:
+You can use the existing `ClientNetworkTransform` that lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`.
 
+Optionally, you can directly add this line to your `manifest.json` file:
 `"com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main"`
+
+You can also just create your own `ClientNetworkTransform` by deriving from `NetworkTransform`, overriding the `OnIsServerAuthoritative` virtual method, and return false: 
```

Comment:
Well... I figure we aren't going to be making many more changes to NetworkTransform at this point and the user's issue was that they had an old version of the ClientNetworkTransform.  When looking for information about "why" in our documentation there was nothing that the user saw as being different (i.e. go download the ClientNetworkTransform here).  _Some users don't create clones and just simply download the branch as a zip, grab what they want, and migrate that into their project._

Since we have no API documentation for NetworkTransform (I was going to point the user to the NetworkTransform API link and then realized there wasn't one), I figured just posting the few lines of code it takes to place a NetworkTransform into owner authoritative mode would be much easier to "spot the things that changed" than for them to re-download what they thought they had already downloaded.

Reply:
I agree with @SamuelBellomo; in-doc samples are difficult to maintain and can easily become outdated. However, it sounds like this could benefit users.

That said, if we do include this excerpt, I suggest rewording it slightly:
> You can also create your own `ClientNetworkTransform` by deriving from `NetworkTransform`, overriding the `OnIsServerAuthoritative` virtual method, and returning false:

Reply:
Ok, both are good suggestions and so I have adjusted the text copy and pointed the example directly to the Packages/com.unity.multiplayer.samples.coop/Utilities/Net/ClientAuthority/ClientNetworkTransform.cs file as opposed to the less manageable embedded code example.